tardis/model/base.py

```diff
@@ -119,13 +118,15 @@ class Composition:
     density : astropy.units.quantity.Quantity
         An array of densities for each shell.
     isotopic_mass_fraction : pd.DataFrame
-    atomic_mass : pandas.core.series.Series
+    atomic_mass : pd.DataFrame
     atomic_mass_unit: astropy.units.Unit
 
     Attributes
     ----------
-    number_density : pd.DataFrame
-        Number density of each isotope in each shell.
+    atomic_mass : pd.DataFrame
+        Atomic mass of elements calculated for each shell.
+    elemental_number_density : pd.DataFrame
+        Number density of each element in each shell.
```

Comment:
Is it really elements we're storing here or individual isotopes?

Reply:
The mass of the individual isotopes are being added up based on their abundances(or mass fractions) and so we are storing the properties for the elements here.